tests/functions_tests/test_poolings.py

```diff
@@ -10,31 +10,32 @@
 from onnx_chainer.testing import test_onnxruntime
 
 
-@testing.parameterize(
-    {'name': 'average_pooling_2d', 'ops': F.average_pooling_2d,
-     'in_shape': (1, 3, 6, 6), 'args': [2, 1, 0]},
-    {'name': 'average_pooling_nd', 'ops': F.average_pooling_nd,
-     'in_shape': (1, 3, 6, 6, 6), 'args': [2, 1, 0]},
-    {'name': 'max_pooling_2d', 'ops': F.max_pooling_2d,
-     'in_shape': (1, 3, 6, 6), 'args': [2, 1, 0]},
-    {'name': 'max_pooling_nd', 'ops': F.max_pooling_nd,
-     'in_shape': (1, 3, 6, 6, 6), 'args': [2, 1, 0]},
-)
+@testing.parameterize(*(testing.product({
+    'name': ['max_pooling_2d'],
+    'in_shape': [(1, 3, 6, 5)],
+    'args': [[2, 2, 0]],
```

Comment:
Is there any particular reason why you changed the arguments in addition to adding `cover_all` option to test?

Reply:
OK, thanks!